packages/grid/src/Grid.tsx

```diff
@@ -1907,6 +1907,13 @@ class Grid extends PureComponent<GridProps, GridState> {
       metrics.rowHeight
     );
 
+    // Check if at the top and attempting to scroll up
+    // Or at the bottom and attempting to scroll down
+    if ((top === 0 && deltaY < 0) || (top >= rowCount - 1 && deltaY > 0)) {
```

Comment:
Something doesn't seem right here that you are introducing a `return` that wasn't there before.

Reply:
Well we return because we are no longer looking to scroll, and thus handle movement of the wheel, if we are attempting to scroll down if we are at the bottom, or scroll up if we are at the top. 

But I am looking into to modify. the logic right now, since as per your observation, the IrisGrid is behaving quite differently then the Grid in terms of the scroll.